sdks/java/extensions/google-cloud-platform-core/src/main/java/org/apache/beam/sdk/extensions/gcp/options/GcsOptions.java

```diff
@@ -53,21 +49,26 @@ public interface GcsOptions extends ApplicationNameOptions, GcpOptions, Pipeline
   /**
    * The ExecutorService instance to use to create threads, can be overridden to specify an
    * ExecutorService that is compatible with the user's environment. If unset, the default is to
-   * create an ExecutorService with an unbounded number of threads; this is compatible with Google
-   * AppEngine.
+   * create a ScheduledExecutorService with a core number of threads equal to
```

Comment:
Since you marked the methods as deprecated, please add the `@deprecated` javadoc comment telling folks to use ExecutorOptions#getScheduledExecutorService

Ditto for set.